src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/TemplateConfigModel.cs

```diff
@@ -415,9 +415,11 @@ private static BaseSymbol SetupDefaultNameSymbol(string? sourceName)
             Dictionary<string, IValueForm> formMap = new Dictionary<string, IValueForm>(StringComparer.Ordinal);
 
             // setup all the built-in default forms.
-            foreach (KeyValuePair<string, IValueForm> builtInForm in ValueFormRegistry.AllForms)
+            // name of the form is form identifier
+            // this is only possible for the forms that don't need configuration
+            foreach (KeyValuePair<string, IValueFormFactory> builtInForm in ValueFormRegistry.FormLookup)
             {
-                formMap[builtInForm.Key] = builtInForm.Value;
+                formMap[builtInForm.Key] = builtInForm.Value.Create();
```

Comment:
this logic is a bit vague and undocumented.
It basically allows to refer to value form based on its identifier; and not the name.
In this case default configuration is used (which doesn't make a lot of sense for configurable forms).
I kept this logic as is, not to introduce breaking change if anyone is using it.